symengine/lib/symengine_wrapper.pyx

```diff
@@ -4507,11 +4507,17 @@ cdef class _Lambdify(object):
     cdef _load(self, const string &s):
         raise ValueError("Not supported")
 
+    cdef void unsafe_real_ptr(self, double *inp, double *out) nogil:
+        raise ValueError("Not supported")
```

Comment:
You need to acquire `gil` before raising an exception.

Reply:
Not required since [cython 0.29](https://cython.readthedocs.io/en/latest/src/changes.html#id28), but it looks like symengine.py supports older versions. I'll fix it.